cogs/randomStuff.py

```diff
@@ -143,6 +143,12 @@ async def reactions(self, ctx):
     async def ban(self, ctx, name):
         await ctx.send(f'Shut up, {name.title()}')
 
+    @commands.command(name='featurerequest', aliases=['request'])
+    async def feature_request(self, ctx, *, text):
+        """Request a new feature for the discord bot!"""
+        #TODO random insults
+        await ctx.send('Add it yourself, bozo! https://github.com/Joelpls/discord-bot')
```

Comment:
Use https://github.com/Joelpls/discord-bot/issues/new?assignees=&labels=&template=feature_request.md&title=

Reply:
I think you sent the wrong link, that doesn't look like an insult library

Reply:
ya make it go to this link, @BradWells